packages/components/src/__future__/Tag/Tag.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react"
+import classNames from "classnames"
+import { Icon } from "@kaizen/component-library/components/Icon"
+import tagIcon from "@kaizen/component-library/icons/tag.icon.svg"
+import { AvatarTag } from "./AvatarTag"
+import { BaseTag } from "./BaseTag"
+import { Dismiss, DismissProps } from "./Dismiss"
+
+/** @todo Add list of all icons names */
+export type TagIcons = "tag"
+
+export type TagColors =
+  | "blue"
+  | "gray"
+  | "green"
+  | "orange"
+  | "purple"
+  | "red"
+  | "yellow"
+
+export interface TagProps {
```

Comment:
This will need to extend the span HTMLElement props

Reply:
Have a look at our SVG component for an example:
https://github.com/cultureamp/kaizen-design-system/blob/main/packages/components/src/SVG/SVG.tsx#L11